src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java

```diff
@@ -293,6 +294,17 @@ public JMenuBar getMenuBar() {
 
         edit.add(fitTreeToScreen);
         fitTreeToScreen.addActionListener((ActionEvent) -> this.fitTreeViewToScreen());
+        {
+            GameBoardFacade.getInstance().getHistory().redo();
+        }
+        if(os.equals("mac")) {
+            redo.setAccelerator(KeyStroke.getKeyStroke('Z', Toolkit.getDefaultToolkit().getMenuShortcutKeyMask() + InputEvent.SHIFT_DOWN_MASK));
+            redo.setAccelerator(KeyStroke.getKeyStroke('Y', Toolkit.getDefaultToolkit().getMenuShortcutKeyMask() + InputEvent.SHIFT_DOWN_MASK));
```

Comment:
Double check if `InputEvent.SHIFT_DOWN_MASK` should be in line 302. I think that might be doing `cmd+Shift+Y` for Mac users, though I'm not entirely sure.

Reply:
I done have the mac near me and I add the ctrl inside.